_posts/2019-01-25-building-apps-with-plug-router.md

```diff
@@ -0,0 +1,354 @@
+---
+author: Sean Callan
+categories: general
+date: 2019-01-25
+layout: post
+title: Building web apps with Plug.Router
+---
+
+When it comes to building a web application with Elixir many people will immediately reach for Phoenix.
+However, did you know `Plug.Router` is just as viable an option?
+Sometimes, it can be even faster.
+
+## The project
+
+For this project we'll build a simple single page portfolio site.
+We can expect our site to load and display our portfolio from a file, database, or somewhere else dynamically. As well as allowing users to submit contact information via a web form.
+
+__Please note__: To keep the application and tutorial concise, we will forego database backing and stub this out in favor of concentrating on the web portions.
+
+Want to skip the reading and just look at a code?
+Head over to [elixirschool/router_example](https://github.com/elixirschool/router_example).
+
+## Getting started
+
+To get started we need to do a couple of things:
+
+1. Generate a new project with `mix new --sup`
+1. Add the `plug_cowboy` dependency to `mix.exs`
+1. Put our router is into the supervision tree of our application
```

Comment:
Typo: Should probable be "Put our router into..."

Reply:
Thank you!